railties/lib/rails/application/configuration.rb

```diff
@@ -287,6 +287,7 @@ def load_defaults(target_version)
             active_record.belongs_to_required_validates_foreign_key = false
             active_record.before_committed_on_all_records = true
             active_record.default_column_serializer = nil
+            active_record.encryption.add_to_filter_parameters = true
```

Comment:
Is the intention to make this the default in new apps? This isn't really a doc change anymore is it?

Reply:
Fixed the doc and I think we can now leave it as the new 7.1 default value.